packages/firebase/webpack.config.js

```diff
@@ -40,7 +40,7 @@ const baseConfig = {
       mangle: {
         props: {
           ignore_quoted: true,
-          regex: /^_[^_]|[^_]_$/
+          regex: /^_[^_][^lat]|[^_]_$/
```

Comment:
Can you please comment this?  Somebody reading this is going to have no idea why this regex is what it is.  And in 2 years we'll have no idea whether we still need "lat" and what might happen if we remove it.

Reply:
Yup, can do.